apps/api/src/app.module.ts

```diff
@@ -76,7 +77,7 @@ const baseModules: Array<Type | DynamicModule | Promise<DynamicModule> | Forward
   InboundParseModule,
   SharedModule,
   HealthModule,
-  EnvironmentsModule,
+  EnvironmentsModuleV1,
```

Comment:
I assumed we want to isolate logic for new dashboard in separate module, e.g. as with workflows, but it could as well be in the same just under different controller, if we want to.